src/include/migraphx/op/multinomial.hpp

```diff
@@ -24,8 +24,8 @@
 #ifndef MIGRAPHX_GUARD_OPERATORS_MULTINOMIAL_HPP
 #define MIGRAPHX_GUARD_OPERATORS_MULTINOMIAL_HPP
 
-#include <migraphx/operation.hpp>
```

Comment:
I'm seeing the same compilation error as Charlie here. Leaving operation.hpp or including reflect.hpp both fix it.